### DISCUSSION TITLE
lingdocdownについてご意見ください！Welcome to lingdocdown Discussions!

### DISCUSSION THREAD

**Discussion by CLRafaelR** [CLRafaelR/lingdocdown#1] [General]
## 👋 lingdownへようこそ！

`lingdown`は，プログラミングによる，言語学向けの文書の生成を支援する説明書およびテンプレートです。とりわけ，R言語の軽量マークアップ言語であるRMarkdownを使って，より簡単に言語学の論文が書けるようにしていきます。

こちらのGitHub Discussionsで，みなさまからご質問・ご意見・アイデアをいただきたいと思います。 ですので，
  * 何か`lingdown`でお困りごとがありましたら，[Q&Aページ](https://github.com/CLRafaelR/lingdocdown/discussions/categories/q-a)にご質問ください
  * 「`lingdown`で，これができたらいいのに」というアイデアがありましたら，[Ideasページ](https://github.com/CLRafaelR/lingdocdown/discussions/categories/ideas)で教えてください
  * 「こんな使い方してみた！」という実例は，[Show and tellページ](https://github.com/CLRafaelR/lingdocdown/discussions/categories/show-and-tell)で，まさにShow and tellしてください！`lingdocdown`でレポート・論文・スライド・ポスター書いてみたという，みなさんからの熱いリンクお待ちしております！

`lingdocdown`がより便利なものになるよう，皆様と一緒に作り上げていきたいです。どうぞお力💪をお貸しください！

**Comment by kishiyamat** [comment]
ほしいものです↓

- 「言語学向けの文書の生成を支援する」ということなので、手っ取り早く「これをこうしたら論文ができる」みたいなテンプレート
- おすすめのディレクトリ構成

**Reply by kishiyamat** [reply]
運用としては

1. Discussionsで必要なものを話し合う
2. 実装のイメージがついたら担当者が Issue を作ってブランチを切って作業（features/template-for-lsj）
3. 作業完了時、コメントが欲しいタイミングで Pull Request を出す
4. 管理者が Approve したら main にマージ

って感じですかね。

**Reply by CLRafaelR** [reply]
@kishiyamat さん

一応，テーマごとにディスカッションの場所を分けました！

[欲しいも🍠のリスト](https://github.com/CLRafaelR/lingdocdown/discussions/3)

4️⃣ ✖️ 3️⃣ 󠀽=[運用案](https://github.com/CLRafaelR/lingdocdown/discussions/5)

**Comment by kishiyamat** [comment]
https://github.com/liao961120/ntuthesis2 これが目指すベースラインですかね

**Reply by CLRafaelR** [reply]
お返事遅くなりすみませんでした。

確かにこのテンプレート、いいですよね～

**Reply by CLRafaelR** [reply]
これを「ベースライン」にするのは目標が高い気もしましたが(笑)、頑張ってやってみましょうか😆👍
